docs/products/clickhouse/concepts/data-integration-overview.md

```diff
@@ -0,0 +1,112 @@
+---
+
+import {ConsoleIcon} from "@site/src/components/ConsoleIcons";
+
+Aiven for ClickHouse® supports different types of integration allowing you to efficiently connect with other services or data sources and access the data to be processed.
+
+There are a few ways of classifying integration types supported in Aiven for ClickHouse:
+
+- [By purpose](/docs/products/clickhouse/concepts/data-integration-overview#observability-integrations-vs-data-source-integrations):
+  observability integration vs data source integration
+- [By location](/docs/products/clickhouse/concepts/data-integration-overview#in-aiven-integrations-vs-external-integrations):
+  in-aiven integration vs external integration
+- By scope: [managed databases integration](/docs/products/clickhouse/concepts/data-integration-overview#managed-databases-integration) and
+  [managed credentials integration](/docs/products/clickhouse/concepts/data-integration-overview#managed-credentials-integration)
+
+## Observability integrations vs data source integrations
+
+Aiven for ClickHouse supports observability integrations and data source integrations,
+which have different purposes:
+
+- [Observability integration](/docs/products/clickhouse/howto/list-integrations) is
+  connecting to other services (either in-Aiven or external) to expose and process logs
+  and metrics.
+- Data service integration is connecting to other services (either in-Aiven or external)
+  to use them as data sources. In Aiven for ClickHouse, data service integration is
+  possible with the following data source types:
+
+  - Apache Kafka®
+  - PostgreSQL®
+  - MySQL®
+  - ClickHouse®
+  - Amazon S3®
+
+## In-Aiven integrations vs external integrations
+
+By enabling data service integrations, you create streaming data pipelines across
+services. Depending on where the services are located, you can have either in-Aiven
+integrations (between Aiven services) or external integrations (between an Aiven service
+and an external data source or application).
+
+For integrating with external data sources, Aiven for ClickHouse provides two types of
+data service integrations:
+
+- [Managed databases](/docs/products/clickhouse/concepts/data-integration-overview#managed-databases-integration)
+- [Managed credentials](/docs/products/clickhouse/concepts/data-integration-overview#managed-credentials-integration)
+
+## Managed credentials integration
+
+The managed credentials integration uses the
+[ClickHouse named collections](https://clickhouse.com/docs/en/operations/named-collections)
+logic. It allows storing key-value pairs that are to be used as credentials when
+connecting to external data sources. To integrate data, you create tables using table
+engines. With the managed credentials integration enabled,
+querying the data is easier and quicker since you no longer need connections parameters in
+each query. They are stored and available from in-Aiven credential storage.
```

Comment:
/related to the comment above/ `from in-Aiven credential storage` definitely sounds strange.